app/controllers/TaxYearRequest.scala

```diff
@@ -48,8 +48,9 @@ abstract class TaxYearRequest @Inject()(
 
   def transformation(implicit request: AuthenticatedRequest[_]): Future[Result] =
     extractViewModel map {
```

Comment:
I think this is only used in `TaxsController` can we move it there and ensure it has the right test coverage?